source/plugin/scheduler.rst

```diff
@@ -93,6 +94,35 @@ To cancel a task, simply call the ``Task#cancel`` method:
 
 	task.cancel();
 
+If you need to cancel the task, from within the runnable itself, you can instead opt to use a `Consumer<task>` in order to access the task from within.
```

Comment:
\`Consumer\<task\>\` --> \`\`Consumer\<Task\>\`\`

Also, "If you need to cancel the task, from within the runnable itself, ..." sounds a bit clunky.
Maybe remove the first comma?
"If you need to cancel the task from within the runnable itself, ..."